internal/identity/independent_system_assigned_user_assigned.go

```diff
@@ -0,0 +1,90 @@
+package identity
+
+import (
+	"fmt"
+
+	msivalidate "github.com/hashicorp/terraform-provider-azurerm/internal/services/msi/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+var _ Identity = IndependentSystemAssignedUserAssigned{}
+
+type IndependentSystemAssignedUserAssigned struct{}
+
+func (s IndependentSystemAssignedUserAssigned) Expand(input []interface{}) (*ExpandedConfig, error) {
```

Comment:
@neil-yechenwei every instance I've seen of this is a Swagger bug - where the Service supports both System Assigned, User Assigned and System Assigned User Assigned but it's not correctly documented - can you confirm with the API/service team here?

Reply:
Sure. Will do.

Reply:
@tombuildsstuff , service team confirmed that they only support either SystemAssigned or UserAssigned or None for Logic Apps. They don’t support both at the same time. So, the swaggers are valid as of now.